filesystems/drs/src/main/scala/cromwell/filesystems/drs/DrsReader.scala

```diff
@@ -96,7 +96,7 @@ case class GcsReader(googleAuthMode: GoogleAuthMode,
         throwable =>
           (requesterPaysProjectIdOption, throwable) match {
             case (Some(requesterPaysProjectId), storageException: StorageException)
-              if storageException.getMessage == "Bucket is requester pays bucket but no user project provided." =>
+              if storageException.getMessage.contains("requester pays bucket but no user project") =>
```

Comment:
I did not find a unit test that covered this.

Reply:
At least this one is checked by the compiler!